src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -264,7 +264,15 @@ def load_default_config(config):
 
         "nfs": ["./scripts/cloud_init_nfs.sh"],
 
-        "lustre_server": ["./deploy/storage/auto_share/setup_lustre.sh"]}
+        "lustre_server": [
+            "./deploy/kubelet/" + config["preworkerdeploymentscript"],
+            "./deploy/kubelet/" + config["postworkerdeploymentscript"],
+            "./scripts/cloud_init_lustre.sh",
+            "./scripts/prepare_lustre_centos.sh",
+            "./deploy/storage/auto_share/setup_lustre.sh",
+            "./deploy/cloud-config/lustre.kubelet.service.template"
+        ]
```

Comment:
please add these to filemap, or I would refactor all of these later.

Reply:
Since adding to filemap also applies to worker, infra, etc., let's refactor all these in a separate PR.